### ISSUE TITLE
Acc of resnet18_dctcn_video_boundary in my test is wrong

### ISSUE THREAD

**Issue by Chengzhig** [mpc001/Lipreading_using_Temporal_Convolutional_Networks#52]
![image](https://user-images.githubusercontent.com/68269246/206637798-19e95b72-997a-429e-b774-1128f6636bc1.png)
I was preprocessed LRW data with /preprocessing/crop_mouth_from_video.py code . then put .npz data with /lipreading/dataloader into model . But I get the result in the image.
Am I forget some steps?

**Comment by mpc001** [comment]
Hello, that's a weird result on the test set. When testing a model without word boundaries, have you encountered a similar issue? By the way, when testing a model that was trained using word boundary information, please remember to include `<ANNONATION-DIRECTORY>`
